examples/hello/bpStepsWriteReadHip/CMakeLists.txt

```diff
@@ -7,19 +7,15 @@ cmake_minimum_required(VERSION 3.12)
 project(ADIOS2HelloBPStepsWriteReadHipExample)
 
 if(NOT TARGET adios2_core)
-  set(_components CXX)
+  find_package(ADIOS2 REQUIRED COMPONENTS CXX)
+endif()
 
-  find_package(hip QUIET)
-  if(hip_FOUND)
-    enable_language(HIP)
-  endif()
+enable_language(HIP)
 
-  find_package(ADIOS2 REQUIRED COMPONENTS ${_components})
-endif()
+# Needed for the hip cmake targets
+find_package(hip REQUIRED)
```

Comment:
You don't need this is Kokkos is build with HIP backend, but I guess in this example we expect a HIP application that is using ADIOS2 with Kokkos so no need to link the application to Kokkos

Reply:
We build it in our Hip build to add coverage to the example so we that can both test that the example is uptodate and that our adios2 for kokkos api does not break.

> 

You don't need this is Kokkos is build with HIP backend, but I guess in this example we expect a HIP application that is using ADIOS2 with Kokkos so no need to link the application to Kokkos

Yep the example is only hip, no kokkos

Reply:
Agree